scripts/setup-env-centos.sh

```diff
@@ -17,14 +20,16 @@ sudo yum install -y \
   golang \
   krb5-devel \
   nodejs \
-  nvidia-docker2 \
+  nvidia-container-toolkit \
   python36 \
   python36-devel
-sudo systemctl start docker
 
 curl -fsSL https://bootstrap.pypa.io/get-pip.py | sudo python3.6
 
 sudo pip3 install virtualenvwrapper
 
-sudo usermod -aG docker $USER
+sudo usermod -aG docker "$USER"
+sudo systemctl enable docker
+sudo systemctl restart docker
+
 echo "source /usr/bin/virtualenvwrapper.sh" >> ~/.bashrc
```

Comment:
This command is not idempotent and is also pretty aggressive IMO.  I would not be flattered if I ran somebody's helper script and it did this without asking me.